5_ACCOUNTS/index.js

```diff
@@ -267,18 +271,21 @@ function getSaldo() {
                 console.log(chalk.bgRed.black(`Esta conta não existe!`));
                 return getSaldo();
             }
-            
-            if(!readPassword(accountName)) {
+
+            readPassword(accountName)
+              .then(() => {
+                const accountData = getAccount(accountName);
+        
+                console.log(
+                    chalk.bgGreen.black(`O saldo da conta ${accountName} é de R$${accountData.balance}.`)
+                );
+
```

Comment:
aqui em vez de fazer 

`if(readPasword(accountName))`

usamos o `.then()` pois vamos esperar o readPassword terminar de executar, o valor que ele passar internamente em `resolve()` recebemos aqui nesse `then` e o `reject` chega no `catch` , por exemplo:

se tivesse la dentro do readPassword for executado `resolve("Senha correta!")`, iriamos receber aqui assim:

```
readPassword(accountName)
  .then((resultado) => {
    console.log(resultado) // "Senha Correta!"
  })
```

e caso for executado o `reject('Senha incorreta!')`
o resultado seria:
```
readPassword(accountName)
  .then((resultado) => {
    // nao executa
  })
  .catch(err => {
    console.log(resultado) // "Senha Incorreta!"
  })
```